applications/StructuralMechanicsApplication/custom_response_functions/adjoint_conditions/adjoint_semi_analytic_base_condition.h

```diff
@@ -361,10 +371,7 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION)  AdjointSemiAnalyticBaseCondi
     ///@name Protected member Variables
     ///@{
 
-    /**
-     * pointer to the primal condition
-     */
-    Condition::Pointer mpPrimalCondition;
+    TPrimalCondition mPrimalCondition;
```

Comment:
to be discussed if this should be a pointer or not

Reply:
I am open for both. 

The reason i went for an instance is that it was done like this here 

(https://github.com/KratosMultiphysics/Kratos/blob/3b012ccb4428206cf570dbccaa7f7dd49bd6d1cb/applications/StructuralMechanicsApplication/custom_response_functions/adjoint_elements/adjoint_solid_element.h#L130 )

and it was easier to construct the primal element in the initializer list of the constructors

Reply:
I decided to keep them as pointers for now. Like this the number of changes in this PR is reduced.